doc/api/webstreams.md

```diff
@@ -148,6 +148,33 @@ The `readableStream.locked` property is `false` by default, and is
 switched to `true` while there is an active reader consuming the
 stream's data.
 
+#### `readableStream.fromWeb(readableStream[,options])`
+
+<!-- YAML
+added: v17.0.0
+-->
+
+* `readableStream` {ReadableStream}
+* `options` {Object}
+  * `encoding` {string}
+  * `highWaterMark` {number}
+  * `objectMode` {boolean}
+  * `signal` {AbortSignal}
+* `Returns`: {stream.Readable}
+
+#### `readableStream.toWeb(writableStream[,options])`
```

Comment:
All of the members should be documented in alphabetical order.

Reply:
sure, I'll arrange them alphabetically